src/Components/Patient/DailyRounds.tsx

```diff
@@ -718,7 +848,10 @@ export const DailyRounds = (props: any) => {
                   </h3>
                   {/*  */}
                   {diagnoses ? (
-                    <EditDiagnosesBuilder value={diagnoses} />
+                    <EditDiagnosesBuilder
+                      value={diagnoses}
+                      key={diagnosisSeed}
```

Comment:
Wasn't the `useEffect` that updates the internal state based on `props.value` sufficient?